# 以自车为原点 station offset [m]
float32 s

# 坡度 [%]
float32 slope

# 曲率 [1/m]
float32 curvature

# 卡车限速 [m/s]
float32 speed_limit
